lms/djangoapps/courseware/module_render.py

```diff
@@ -776,12 +781,6 @@ def rebind_noauth_module_to_user(module, real_user):
     system.set('user_is_beta_tester', CourseBetaTesterRole(course_id).has_user(user))
     system.set('days_early_for_beta', descriptor.days_early_for_beta)
 
-    # make an ErrorBlock -- assuming that the descriptor's system is ok
```

Comment:
This comment doesn't looks relevant as we already do access `descriptor` properties above. I have removed it but fine with adding it back if needed!